src/ucp/rndv/rndv_am.c

```diff
@@ -16,7 +16,8 @@ static void ucp_rndv_am_probe_common(ucp_proto_multi_init_params_t *params)
     ucp_context_h context = params->super.super.worker->context;
 
     if (!ucp_proto_rndv_op_check(&params->super.super, UCP_OP_ID_RNDV_SEND,
-                                 0)) {
+                                 0) ||
+        (context->config.ext.max_rndv_lanes == 0)) {
```

Comment:
we already set params->max_lanes = context->config.ext.max_rndv_lanes, isn't it enough?
we should make ucp_proto_multi_probe don't add the protocol when it cannot find lanes, including the case params->max_lanes is 0

Reply:
this case is treated as a bug in `ucp_proto_multi_init`:
`ucs_assert(params->max_lanes >= 1);`

should we cancel this behavior?

Reply:
yes, remove the assert